review.md

```diff
@@ -0,0 +1,56 @@
+# Code Review principles and tips
+
+## Why do we code review?
+
+1. Code reviews are essential to improve the level of our code, and to prevent bugs. 
+
+2. A great and fast way for both the reviewer and the code owner to share ideas, learn, and improve their code writing and understanding skills.
+
+3. Allow us as reviewers to get to know parts of the code base we usually don't work on.
+
+4. Help our fellow team members to validate their code quality and push updates faster.
+
+## Tips for reviewing
+
+-   It is important to maintain a healthy, respectful, and civil environment. Feel free to ask questions, offer suggestions, or provide criticism, regardless of your position or level of expertise. However, it is crucial to avoid personal attacks and focus on reviewing the code rather than criticizing the individual who wrote it. Additionally, please refrain from using capital letters, using inappropriate language, or addressing others directly.\
```

Comment:
```suggestion
-   It is important to maintain a healthy, respectful, and civil environment. Feel free to ask questions, offer suggestions, or provide criticism, regardless of your position or level of expertise. However, it is crucial to avoid personal attacks and focus on reviewing the code rather than criticizing the individual who wrote it. Additionally, please refrain from using capital letters (all caps), using inappropriate language, or addressing others directly.

```

Reply:
What does `addressing others directly` mean?

Reply:
Using "You", for instance. The examples put more focus on it.

Reply:
done

Reply:
> Using "You", for instance. The examples put more focus on it.

I think I see what you mean. We should talk about the code rather than who wrote it, but I think this expectation is a bit extreme.

Reply:
I thought the examples would make the expectation clearer... how would you rephrase it? I still think creating a connection between the way the code is written and who wrote it, in a direct way, as shown in the examples, is something we should be thoughtful of.

Reply:
I think we already mention we should focus on the code rather than the individual. I think that if you drop this part here and leave the examples as-is, it should be enough.